.circleci/config.yml

```diff
@@ -42,6 +42,41 @@ commands:
                 command: carthage bootstrap --platform all --cache-builds --configuration Debug --use-xcframeworks   
 
 jobs:
+  detect-breaking-changes:
+    macos:
+      xcode: 13.3.0
+    steps:
+      - checkout
+      - install-mapbox-token
+      - run: 
+          name: "Unzip baseline interface"
+          command: |
+            cd swift-package-baseline
+            unzip baseline-interface.zip
```

Comment:
Don't think zipping is a good idea. If we keep only the latest baseline, this file can be updated (move + several changes for each baseline update).

Reply:
The baseline isn't supposed to be changed frequently. Only once per major release. So, if we analyse the size:
- raw: 1.6 MB
- zipped: 74 KB.
- Number of major release before zipped content will become more than first raw (excluding the size increase with raw diffs): 21 Major release.
- Estimated number of years before the parity reached assuming 1 major release per 2 years: 42 years.
- Probability that library will reach 21st major release: ~0

So I would keep zipped content because it has immediate effect on users network traffic consumption.

Reply:
Large binary files would still have an immediate impact on the time it takes to clone the repository, something SPM does with abandon. Can we host this file anywhere else – say, as an attachment to a release?

Reply:
I don't think it worth it moving baseline out of the repository. For comparison, one snapshot image in the mapbox-navigation-ios can be bigger than this zipped content. Maps team also stores baseline in the repo zipped: https://github.com/mapbox/mapbox-maps-ios/tree/main/scripts/api-compatibility-check

Reply:
What if we run a separate job or update this one and cache baseline on CI?

And it's not true that the baseline is changed on major releases only. We should update it whenever we add new APIs. So it could be that we need to update the baseline for each `rc.1` (and maybe other RCs and GA if they introduce a new API).

Storing baseline in GitHub release artifacts is an interesting idea. It can be automated later, but it's an easy release step that we should have anyway (it's either updating it within the repo or updating the release on Github). And the CI script is not going to be modified as we can read the tag from `swift-package-baseline/baseline.txt`.

Reply:
> And it's not true that the baseline is changed on major releases only.

I mean you don't to update it on each API change. In ideal world it should point to first major release and that's it. 

If you are so agains this baseline caching, I can remove it. It is not necessary for the tool to work. Just speeds things up a bit.

Reply:
My point is that if we use `2.0.0` as a baseline, then add a new API in `2.1.0` and break it in `2.2.0`, it's a breaking change for `2.1.0`, but not for `2.0.0`. That's why we should use the latest RC or GA as a baseline in this tool.

Reply:
Fair point, this will need a redesign of this PR. Coming.

Reply:
PR reworked:
- README.md updated with new instructions.
- update-version.sh script updated with automatic baseline bump during rc.1 release.
- Baseline isn't cached any more.
- Baseline is updated during each RC release.